### PR TITLE
[W5.8c][F11-A1] Tan Ci Kang

### PR DESCRIPTION
**Pull request by ck94rec**

### CODE REVIEW

**Comment by Sheikh-Umar** [comment]
Reviewed.

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -69,3 +69,8 @@
 * Problem: Test fails during the very first time.<br>
   Solution: The output of the very first test run could be slightly different because the program
   creates a new storage file. Tests should pass from the 2nd run onwards.
+
+### Non-Functional Requirements
+* AddressBook should cater to both people proficient in IT and those that do not know about coding.
+* AddressBook can remain open for long period of time without taking up too much of processing power. 
+* Confidential Information in Addressbook should not be easily accessible.
```
Comment (karrui) [inline doc/DeveloperGuide.md:76]: Too vague - think more in depth and decide if this is an NFR or FR. (e.g., provide a password/login mechanism to make it secure is a functionality)

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -69,3 +69,8 @@
 * Problem: Test fails during the very first time.<br>
   Solution: The output of the very first test run could be slightly different because the program
   creates a new storage file. Tests should pass from the 2nd run onwards.
+
+### Non-Functional Requirements
+* AddressBook should cater to both people proficient in IT and those that do not know about coding.
+* AddressBook can remain open for long period of time without taking up too much of processing power.
```
Comment (karrui) [inline doc/DeveloperGuide.md:75]: Be as precise as possible. What does "long period of time" mean? What is "too much of processing power"? How much is too much?

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -69,3 +69,8 @@
 * Problem: Test fails during the very first time.<br>
   Solution: The output of the very first test run could be slightly different because the program
   creates a new storage file. Tests should pass from the 2nd run onwards.
+
+### Non-Functional Requirements
+* AddressBook should cater to both people proficient in IT and those that do not know about coding.
```
Comment (karrui) [inline doc/DeveloperGuide.md:74]: EDIT: On second thoughts, this works as an NFR! :)

**Review by karrui — COMMENTED** [review]
Some comments added. Please close PR after reading! :)

**Pull request closed without merging**